composer.json

```diff
@@ -1,17 +1,36 @@
 {
-	"name": "silverstripe/multiform",
-	"description": "SilverStripe forms with multiple steps, flow control and state persistence",
-	"type": "silverstripe-module",
-	"keywords": ["silverstripe", "forms"],
-	"authors": [{
-		"name": "Ingo Schommer",
-		"email": "ingo@silverstripe.com"
-	}, {
-		"name": "Sean Harvey",
-		"email": "sean@silverstripe.com"
-	}],
-	"require": {
-		"silverstripe/framework": "~3.2"
-	},
-	"license": "BSD-3-Clause"
+    "name": "silverstripe/multiform",
+    "description": "SilverStripe forms with multiple steps, flow control and state persistence",
+    "type": "silverstripe-module",
+    "keywords": [
+        "silverstripe",
+        "forms"
+    ],
+    "authors": [
+        {
+            "name": "Ingo Schommer",
+            "email": "ingo@silverstripe.com"
+        },
+        {
+            "name": "Sean Harvey",
+            "email": "sean@silverstripe.com"
+        }
+    ],
+    "require": {
+        "silverstripe/framework": "^4@dev"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7",
+        "squizlabs/php_codesniffer": "^3.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "SilverStripe\\MultiForm\\": "src/",
+            "SilverStripe\\MultiForm\\Extensions\\": "src/Extensions/",
+            "SilverStripe\\MultiForm\\Models\\": "src/Models/",
+            "SilverStripe\\MultiForm\\Tasks\\": "src/Tasks/",
```

Comment:
We shouldn't need to define all of these sub namespaces individually - the first line should catch them all. You'll also need to add `"minimum-stability": "dev"` and `"prefer-stable": true`, or it will fail in Travis